stallman/formacao-academica.md

```diff
@@ -1,4 +1,48 @@
 [Voltar](intro.md)
 
-Formação acadêmica
-====
\ No newline at end of file
+  
+
+# *Formação acadêmica*
+
+Richards se formou em Física na Universidade de Harvard(1971- 1974).[^1]
+
+Se formou em Física pela Universidade de Harvard, no ano de 1974, onde após se formar, ele estudou programação de computadores no Massachusetts Institute of Technology (MIT) onde se tornou lider da comunidade hacker.[^2]
```

Comment:
Colocar palavras estrangeiras em itálico, tipo: _Massachusetts Institute of Technology_ (MIT)

Reply:
As alterações de palavras estrangeiras foi realizada

`*Free Software Fundation*`